docker-compose.yml

```diff
@@ -1,13 +1,7 @@
 services:
   rfswarm_agent:
-    build:
-      context: .
-      dockerfile: agent/rfswarm-base-image/Dockerfile
+    image: ghcr.io/esthenia-collaboration/rfswarm-base-image:latest
```

Comment:
I don't think using this image is a good idea because:
- I don't think it's getting updated in future (see change in https://github.com/Esthenia-collaboration/rfswarm-docker/pull/20/files)
- even if it was being updated you'd need to make sure this test ran after it as updated, but wouldn't that defeat the point of testing? I think it's better to test the docker file first then make the publish steps not run if the tests failed, but this point opens up a bigger discussion about the workflows and what you and @Maresther-B  want them to do.

Reply:
Hello Dave,
* The use of a GitHub registry is temporary because I'm based on the main branch.
* I agree for this point. However, if we want to check the build before the push it's better to use docker build (as in the publish action) and not docker compose build to reproduce the image exactly. For the Docker Compose check, it can be interesting to test the docker-compose.yml file that we propose for the users with image pulling as we did.